tf-modules/cumulus/variables.tf

```diff
@@ -544,3 +544,9 @@ variable "ecs_custom_sg_ids" {
   type = list(string)
   default = []
 }
+
+variable "deploy_cumulus_distribution" {
+  description = "If true, does not deploy the TEA distribution API"
+  type        = bool
+  default     = true
```

Comment:
We want to default it to false if TEA is the default, so the current user won't need to pass/set this variable.

Reply:
Yeah hmm that's right. From a user perspective it's `false` as the template defaults to TEA. I was thinking `true` because example defaults to cumulus distribution but that shouldn't matter. I see your point.

https://github.com/nasa/cumulus/pull/2353/commits/3e00220523bad590aa8b1e923bf0507b183e4726